bin/runtime/src/lib.rs

```diff
@@ -889,6 +929,51 @@ impl_runtime_apis! {
                 .collect::<Result<Vec<AlephId>, AlephApiError>>()
         }
     }
+
+    impl pallet_contracts_rpc_runtime_api::ContractsApi<Block, AccountId, Balance, BlockNumber, Hash> for Runtime {
+
+        fn call(
+            origin: AccountId,
+            dest: AccountId,
+            value: Balance,
+            gas_limit: u64,
+            storage_deposit_limit: Option<Balance>,
+            input_data: Vec<u8>,
+        ) -> ContractExecResult<Balance> {
+            Contracts::bare_call(origin, dest, value, gas_limit, storage_deposit_limit, input_data, true)
```

Comment:
I would extract this enigmatic `true`s into some constant, like `DEBUG_CONTRACTS`

Reply:
Done